__tests__/fizz-buzz-tree.test.js

```diff
@@ -0,0 +1,15 @@
+const { BinaryTree } = require('../challenges/tree/tree');
+const { fizzBuzzTree } = require('../challenges/fizzbuzz-tree/fizz-buzz-tree');
+
+describe('FizzBuzz Tree', () => {
+  it('determines if value of each node is divisble by 3, 5, or both', () => {
+    const tree = new BinaryTree();
+    tree.add(5);
+    tree.add(15);
+    tree.add(3);
+
+    expect(fizzBuzzTree(tree.root.left)).toBe('Fizz');
```

Comment:
the goal is to have a function that you pass your whole tree to and then it returns the fizzbuzz values - for example: 
```expect(fizzBuzzTree(tree)).toBe(['Fizz', 'FizzBuzz', 'Buzz'])```